### DISCUSSION TITLE
Overwritting villager appearance with resource pack

### DISCUSSION THREAD

**Discussion by mad-depot** [Luke100000/minecraft-comes-alive#915] [General]
Hello. i was wondering if there was a way to force the mod to not load its custom models/sounds if I have a resource pack that replaces villagers active.

**Comment by Luke100000** [comment]
https://github.com/Luke100000/minecraft-comes-alive/wiki/Custom-Skins

**Reply by ZewenSenpai** [reply]
I posted a bug report about this but essentially the example file itself does not seem to work on 1.20.1 even after converting pack data for 15 for the 1.20.1 version.

Here was the post: https://github.com/Luke100000/minecraft-comes-alive/issues/924#issue-2297429190
